server/config/config.go

```diff
@@ -488,6 +476,12 @@ type ScheduleConfig struct {
 	SplitMergeInterval typeutil.Duration `toml:"split-merge-interval,omitempty" json:"split-merge-interval"`
 	// EnableOneWayMerge is the option to enable one way merge. This means a Region can only be merged into the next region of it.
 	EnableOneWayMerge bool `toml:"enable-one-way-merge,omitempty" json:"enable-one-way-merge,string"`
+	// EnableCrossTableMerge is the option to enable cross table merge. This means two Regions can be merged with different table IDs.
+	// This option only works when merge strategy is "table".
+	EnableCrossTableMerge bool `toml:"enable-cross-table-merge,omitempty" json:"enable-cross-table-merge,string"`
+	// MergeStrategy is option to control the merge strategy.
+	// There are some strategics supported: ["table", "raw", "txn"], default: "table"
+	MergeStrategy string `toml:"merge-strategy" json:"merge-strategy,string"`
```

Comment:
em, maybe something like `cluster-type`? and move it to uppder level? (in Config not in ScheduleConfig)

Reply:
I don't think the "table" belongs to `cluster-type`. Maybe we need a better classification? But we cannot find a good one.

Reply:
or maybe `cluster-key-type`? I think it should not be restricted to merge feature. Other feature relies on it too.

Reply:
How about moving it into `PDServerConfig`?

Reply:
I think it's a good idea.